From jrm8005@gmail.com  Wed Nov 12 14:36:41 2008
From: "J.R. Mauro" <jrm8005@gmail.com>
Date: Fri, 7 Nov 2008 17:49:07 -0500
Subject: Staging: otus: fix name clash
To: Greg KH <gregkh@suse.de>
Message-ID: <20081107224907.GC3353@tuxbook-pro>
Content-Disposition: inline


Make p80211wext_handler_def in the otus driver static. It's not used outside
this file and causes name clashes if it's visible.

Signed-off by: J.R. Mauro <jrm8005@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/otus/usbdrv.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/staging/otus/usbdrv.c
+++ b/drivers/staging/otus/usbdrv.c
@@ -287,7 +287,7 @@ static const iw_handler usbdrv_private_h
     NULL,               /* SIOCIWFIRSTPRIV */
 };
 
-struct iw_handler_def p80211wext_handler_def = {
+static struct iw_handler_def p80211wext_handler_def = {
     .num_standard = sizeof(usbdrvwext_handler) / sizeof(iw_handler),
     .num_private = sizeof(usbdrv_private_handler)/sizeof(iw_handler),
     .num_private_args = sizeof(usbdrv_private_args)/sizeof(struct iw_priv_args),
@@ -948,7 +948,6 @@ int zfLnxUnregisterVapDev(struct net_dev
 #endif
 
 //extern void zfiWlanQueryMacAddress(zdev_t* dev, u8_t* addr);
-extern struct iw_handler_def p80211wext_handler_def;
 
 extern int usbdrv_open(struct net_device *dev);
 extern int usbdrv_close(struct net_device *dev);
